src/main/java/com/airlines/dao/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,106 @@
+package com.airlines.dao;
```

Comment:
you could leave it as is, but also you could make package impl and store your implementations there.
Interfaces just be as they are - in dao

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/8f61d1a09ce2d8aa9876fbe7bcce6e05b7a32a08